### PR TITLE
DOCS-2182 Added link to Configuring Logging

### PR DESCRIPTION
**Pull request by valkyrie69**
And other updates

### CODE REVIEW

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 1]
```diff
@@ -1,71 +1,53 @@
...
-:keywords: cloudhub, managing, monitoring, deploy, runtime manager, arm
 
-image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-active.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
+After you deploy apps, use the Runtime Manager console to monitor and manage running applications. The different management functionalities available depend on the xref:deployment-strategies.adoc[deployment strategy] you use. 
 
-After you deploy an application through Runtime Manager, you can use the console to monitor and manage it while running. Depending on the xref:deployment-strategies.adoc[deployment strategy] used (where the application is deployed to and from what console), there is a different set of possible actions you can perform.
+The basic management tools in the Runtime Manager console allow you to perform the following tasks :
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: allow > enable

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 2]
```diff
@@ -1,71 +1,53 @@
...
-After you deploy an application through Runtime Manager, you can use the console to monitor and manage it while running. Depending on the xref:deployment-strategies.adoc[deployment strategy] used (where the application is deployed to and from what console), there is a different set of possible actions you can perform.
+The basic management tools in the Runtime Manager console allow you to perform the following tasks : 
 
-[TIP]
-Whether you're using the https://anypoint.mulesoft.com[cloud based] or the Private Cloud Edition of the Anypoint Platform, the features described in this page work identically in both. If you’re interested in requesting the Anypoint Platform Private Cloud Edition, please contact your sales representative.
-
-This page introduces the basic management tools in the console that are common to both types of deployments:
+* <<start_stop_delete_app, Start, stop, and delete apps>>
+* <<status_states, View status states for apps>>
+* Update your apps
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: This is the only list item that uses "your" before "apps"; standardize one way or the other.

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 3]
```diff
@@ -1,71 +1,53 @@
...
+* Monitor and troubleshoot apps
 
-* <<Start, Restart, Stop, and Delete an Application>>
-* <<Status States>>
-* <<Updating Your Application>>
-* Various xref:monitoring.adoc[monitoring] and xref:troubleshooting.adoc[troubleshooting] functionalities
 
+For applications deployed to xref:managing-applications-on-your-own-servers.adoc[your own servers] you can also:
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: Add comma after "your own servers."

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 4]
```diff
@@ -1,71 +1,53 @@
...
+* xref:managing-servers.adoc[Manage servers]
+* xref:server-group-create.adoc[Create clusters and server groups]
+* xref:sending-data-from-arm-to-external-analytics-software.adoc[Send data from Runtime Manager to external analytics software]
+* xref:flow-management.adoc[Manage individual flows]
 
-* xref:managing-servers.adoc[Manage Servers]
-* xref:managing-servers.adoc[Create Clusters and Server Groups]
-* xref:sending-data-from-arm-to-external-analytics-software.adoc[Send Data from Runtime Manager to External Analytics Software]
-* xref:flow-management.adoc[Manage Individual Flows]
+On applications deployed to xref:managing-applications-on-cloudhub.adoc[CloudHub] you can also:
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: Add comma after CloudHub.

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 5]
```diff
@@ -1,71 +1,53 @@
...
 
-On applications deployed to xref:managing-applications-on-cloudhub.adoc[CloudHub] you can also manage the following elements:
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc:32]: Comma, same location, throughout intro sentences

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 6]
```diff
@@ -1,71 +1,53 @@
...
-
-* xref:notifications-on-runtime-manager.adoc[Notifications]
-* xref:insight.adoc[Insight]
-* xref:managing-queues.adoc[Queues]
-* xref:managing-schedules.adoc[Schedules]
-* xref:secure-application-properties.adoc[Secure Application Properties]
+* xref:notifications-on-runtime-manager.adoc[Manage notifications]
+* xref:insight.adoc[Use Insight to troubleshoot business transactions and events on your Mule apps]
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc:26]: I cannot find "Insight" as a product name in the list that we currently use (from Sanjna), although there is a note to not use "insights & analytics" in place of Anypoint Analytics.
However, when I perform internet search, I do see refs. to CloudHub Insight and Insight in Mule. So...your discretion. If there is a more full product name of which you are aware, consider using it here.

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 7]
```diff
@@ -1,71 +1,53 @@
...
-Sign into Anypoint Platform and click *Runtime Manager*:
-
-image::deploying-to-your-own-servers-get-to-arm.png[]
-
-Runtime Manager has two main tabs--*Applications* and *Servers*.
-
 === Applications Tab
 
-In Runtime Manager, an application is a Mule Application that is running on a Mule runtime or an API gateway runtime instance. The instance can be running in CloudHub, in an on-premise server of your organization, in a group of servers, or in a cluster.
+In Runtime Manager, an application is a Mule application that is running on a Mule runtime engine (Mule) or an API gateway runtime instance. The instance can be running in CloudHub, in an on-premises server of your organization, in a group of servers, or in a server cluster.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: "an application is a Mule application" sounds odd. Can we put the first "application" in quotation marks or otherwise rephrase so that it is clear that what is **called** an "application" in Runtime Manager is a Mule application that is..."?

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 8]
```diff
@@ -1,71 +1,53 @@
...
 
 In the *Applications* menu you can get a quick view of the applications that are currently deployed and running for your account. Click an entry to display additional information (as shown below), or click *Manage Application* to see the application's xref:monitoring-dashboards.adoc[dashboard] and settings page:
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: "below" > "next" or "here"

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 9]
```diff
@@ -1,71 +1,53 @@
...
 
-[TIP]
-For server groups, an application deployed to the group runs on every server in the group.
+An application deployed to a server group runs on every server in the group.
 
 === Servers Tab
 
 [NOTE]
-This tab is only relevant to deploying to servers on-premises. CloudHub workers aren't displayed here.
+This tab is only used to deploy apps to on-premises servers. CloudHub workers aren't displayed here.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: "only used" > "used only"

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 10]
```diff
@@ -1,71 +1,53 @@
...
 
 In Runtime Manager, a server is a Mule or API gateway instance in a location other than CloudHub (on-premises or in a different cloud server) that you must first register in the Runtime Manager console. Once registered and configured in this tab, you can deploy your applications to it through the *Applications* tab.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: Ditto re. making clear that "server" is being used as a term here rather than as an object or concept.

"Once registered..., you can" is a misplaced modifier, implying that once YOU are registered..."
Reword to "After you register and configure your server, you can deploy your applications to it...."

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 11]
```diff
@@ -85,42 +67,43 @@ From the *Servers* tab, select one of the following options to perform these tas
 
 For more about the options on this tab, see xref:managing-servers.adoc[Managing Servers].
 
-=== Switching Environments
+=== Switch Environments
 
-After you have multiple environments in your organization, you can switch between them. From any screen in Runtime Manager, you can switch to a different environment by clicking the current environment, displayed in the upper left corner of the screen, and picking another environment.
+If you have multiple environments in your organization, you can switch between them.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: ...between them.  >  ...between them:     (colon leading to first step)

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 12]
```diff
@@ -85,42 +67,43 @@ From the *Servers* tab, select one of the following options to perform these tas
...
-. Select the environment to switch to.
-. Click the *switch* button.
+image::managing-deployed-applications-0e10f.png[]
 
+[[start_stop_delete_app]]
 == Start, Restart, Stop, or Delete an Application
 
-To start, restart, stop, or delete an application, click an application, and click the status down-arrow (the menus differ by Status):
-
+. In *Applications*, click an application to open its management panel on the right side of the screen.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: click an application > click an application name?

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 13]
```diff
@@ -85,42 +67,43 @@ From the *Servers* tab, select one of the following options to perform these tas
...
-
-Click *Start*, *Restart*, *Stop*, or *Delete* as needed.
-
++
+. Click the action to perform: + 
 * *Start* - Starts stopped applications
-* *Restart* - Resets applications that have already started, and if the application is undeployed, sends the application's ZIP file to the server to start the application on the server
-* *Stop* - Stops running applications
-* *Delete* - Deletes the application
+* *Restart* - Resets applications that have already started. If the application is undeployed, the application's ZIP file is sent to the server to start the application on the server.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: I suggest enhancing the bullet list's scannability by changing the presentation slightly:

  o Restart
     Resets applications....

  o Stop
     Stops running applications.

 o Delete
    Deletes the application.
   


GLOBAL consideration for definition lists (example in https://docs.mulesoft.com/general/getting-started/glossary, although I recommend against the colon)

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 14]
```diff
@@ -85,42 +67,43 @@ From the *Servers* tab, select one of the following options to perform these tas
...
 
 [IMPORTANT]
 On CloudHub, after you delete an application, your log data is no longer accessible through the console. CloudHub archives old log data for a limited period of time before it is purged. This allows you to recover the data if needed. Open a support case at: https://support.mulesoft.com[https://support.mulesoft.com] for more information.
 
 [NOTE]
 ====
-For applications deployed through PCF:
+For applications deployed through Pivotal Cloud Foundry:
 
 * *Restart* - Redeploys your application on the virtual machine.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: ditto bullet list formatting suggestion; GLOBAL (I will stop marking)

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 15]
```diff
@@ -85,42 +67,43 @@ From the *Servers* tab, select one of the following options to perform these tas
...
 * *Stop* - Destroys the virtual machine running the application but retains the entry on Runtime Manager to keep log continuity.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: keep > maintain

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 16]
```diff
@@ -85,42 +67,43 @@ From the *Servers* tab, select one of the following options to perform these tas
...
 * *Undeploy* - Destroys the virtual machine running the application and deletes the entry for it on Runtime Manager.
 ====
 
-
+[[status_states]]
 == Status States
 
 The following states appear in the *Status* column:
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: Looking at this entire section, I suggest that we add some text that introduces the status categories--application, replica, server, server groups, and cluster--and then follow with the titles, an intro sentence defining the category further, if necessary, followed finally by the accompanying table or list.

Ditto previous list format suggestion.

Evaluate tables to determine whether they actually need to be tables rather than definition lists.

Remove title and number from first table.

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 17]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
 | Partial (x of y running) with communication issue - some nodes are running some are down, visibility issues between them
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: This table is a little unusual because it comprises a column of icons and a column of definition-list-type entries.
I wonder if the table could be icon + definition, without the beginning term?

I also wonder if we should look at standardizing all of this UI documentation to list format.

I further wonder if we NEED to document the UI? The user tasks that document using the UI might be sufficient.

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 18]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 
-== Applications Dashboard
+== Applications Management Panel
 
 If you select any of the applications on the *Applications* tab, a panel opens on the right. Here you can see additional information about the application and access its settings.
 
 image::dashboard-ch.png[dashboard]
 
-What this dashboard displays depends on if your application is deployed on xref:managing-applications-on-cloudhub.adoc[CloudHub] or on xref:managing-applications-on-your-own-servers.adoc[Mule]. You will always see the status, the time of the last modification, and the deployed application ZIP file. You can click the *Choose File* button to change the application ZIP file.
+What the management panel displays depends on if your application is deployed on xref:managing-applications-on-cloudhub.adoc[CloudHub] or on xref:managing-applications-on-your-own-servers.adoc[your own servers]. You always see the status, the time of the last modification, and the deployed application ZIP file. Click the *Choose File* button to change the application ZIP file.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: if > whether

Note that there is a hidden task here in the final sentence. This is a case in which I would reword to use "can": "You can click Choose File if you want to change the... " or "If you want to change...you can...." We are not directing the audience to do this right now, in a flow of steps.

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 19]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 
-Applications on CloudHub display info about the CloudHub worker they run on.
+Applications on CloudHub display info about the CloudHub worker on which they run.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: info > information

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 20]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 
 Applications deployed on clusters and server groups display the list of servers and the status of each.
 
-You can click the *Manage Application* button to access a menu where you can view and configure several more application settings:
+Click *Manage Application* to access a menu where you can view and configure several more application settings:
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: Ditto previous comment about the wording of hidden tasks. It would be best if they were not hidden at all, but independent or part of a larger task, with steps. In conceptual narrative, I suggest rewording to indicate ability or option ("can"), which might be most palatable in the form of "If you want to..."

     "If you want to view and configure more applications, you can do so from the menu that results from clicking....

     "Clicking Manage Application shows a menu that enables you to make choices to view and configure....

"a menu where you can" > Menu enable you to make choices; they are not platforms for viewing and configuring.
Reword for precision.

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 21]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 
 image::managing-deployed-applications-38b94.png[]
 
-From there you can also view the *Dashboad* to see the full detail dashboard of your application. See xref:monitoring-dashboards.adoc[Monitoring Dashboards] for more info about what you can see here.
+In the app's settings page, you can view the *Dashboad* to see the full detail dashboard for your application. See xref:monitoring-dashboards.adoc[Monitoring Dashboards] for more info about what you see on the app's full detail dashboard.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: These two sentence are largely repetitive, esp. wordings like "view the Dashboard to see the full detail dashboard...." Consider reducing the detail of each so that they complement each other:

"In the app settings page you can see full details of your application by viewing Dashboard. See Monitoring Dashboards."

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 22]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 
 image::managing-deployed-applications-107ad.png[]
 
 
-You can click the *Insight* button to see data at a transaction level. See xref:insight.adoc[Insight] for more information about what you can see here.
+Click the *Insight* button to see data at a transaction level. See xref:insight.adoc[Insight] for more information.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: Ditto previous suggestions about rewording hidden tasks.

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 23]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 
 image::managing-deployed-applications-76124.png[]
 
 [NOTE]
-The dashboard and insights are features that are not currently available for the Anypoint Platform Private Cloud Edition. See xref:deployment-strategies.adoc[deployment strategies].
+The dashboard and insights features are not currently available for Anypoint Platform Private Cloud Edition. See xref:deployment-strategies.adoc[deployment strategies].
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: I think it is odd to have the image followed directly by a Note. Just plain text will do.

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 24]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 
-== Monitoring Applications
+== Monitor Applications
 
-Depending on the xref:deployment-strategies.adoc[deployment strategy] used (where the applications are deployed to, and through what console). Different tools exist for monitoring events that occur on your application as well as the performance of the servers or virtual servers it runs on. See xref:monitoring.adoc[Monitoring Applications] for more details.
+Depending on the xref:deployment-strategies.adoc[deployment strategy] used, different tools exist for monitoring events that occur on your application, as well as the performance of the servers or virtual servers it runs on. See xref:monitoring.adoc[Monitoring Applications] for more details.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: Consider rewording for active voice and direct address: "The tools that you can access to monitor application events and server or virtual server performance depend on your deployment strategy. See...."

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 25]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 
 
-== Updating Your Application
+== Update Your Application
 
 If you made changes to your applications and want to upload a new version:
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: applications > application

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 26]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 
-. Click *Choose file* on the Deployment screen for that application. +
-The new filename appears in italicized text.
+. Click *Choose file* on the deployment screen for that application.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: You can omit "for that application."

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 27]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
+. Select either *Import file from Exchange* or *Upload file*.
 . Click *Apply changes* to use the new file for deployment. +
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: You can omit "to use the new file for deployment."
Note that the introduction refers to "upload" rather than "deploy."

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 28]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 Within a few seconds, your application redeploys. While redeploying, the application status indicator changes to blue, and then turns green after the deployment completes.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc:288]: Is this text really necessary? Maybe reduce to "The application status indicator turns green when deployment is complete."

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 29]
```diff
@@ -205,51 +188,51 @@ Replica statuses include:
...
 . For applications deployed to CloudHub, you can click *Logs* to see a live redeployment of your application.
 
 [TIP]
-If you're deploying your application to a cluster on-premises, Runtime Manager installs the new app version progressively on each server to keep the service working with no downtime throughout the process.
+If you're deploying your application to an on-premises cluster, Runtime Manager installs the new app version progressively on each server to keep the service working with no downtime throughout the process.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: "...installs the new...process."  >  "...installs the new version progressively on each server to ensure no downtime."

File: modules/ROOT/pages/managing-deployed-applications.adoc [hunk 30]
```diff
@@ -259,20 +242,16 @@ You can set up email alerts that are sent whenever certain events occur to your
 
 All users of Anypoint Platform, even those without permissions to create alerts, can switch the alerts that are already created into an `active` or `inactive` state for their user. This determines what email alerts are sent to them.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-deployed-applications.adoc]: Suggestion:

All users of Anypoint Platform can switch existing email alerts between 'active' or 'inactive' for their customers.

If "customers" is not correct, try to find some alternative to repeating "users."
If you keep the added clarification, be sure to make clear to whom "them" refers.

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 31]
```diff
@@ -2,53 +2,40 @@
...
-image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
 CloudHub provides access to log data that includes deployment messages and events for each worker.
 
-
 [NOTE]
 ====
-Logs and diagnostics are currently available only for applications deployed to CloudHub workers. In cases where applications are deployed using the Runtime Manager local server, you can send data xref:sending-data-from-arm-to-external-analytics-software.adoc[to External Analytics Software]. See xref:deployment-strategies.adoc[Deployment Strategies] and xref:monitoring.adoc[Monitoring Applications] for more details.
+Logs and diagnostics are currently available only for applications deployed to CloudHub workers. In cases where applications are deployed to a local server, you can send data to external analytics software.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: In cases where > When (or If)

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 32]
```diff
@@ -2,53 +2,40 @@
...
 
 == View Logs
 
-. In Runtime Manager, click *Applications* in the left menu, then click the application that is deployed to CloudHub to open its management options.
-. In the left menu, click *Logs*. The application logs are displayed.
-+
-//image::elmstart.png[ELMStart]
+View logs for applications deployed to CloudHub.
 
+. In Runtime Manager, click *Applications* in the left menu, then click the application that is deployed to CloudHub to open the management panel on the right.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: Is the "left menu" the navigation pane?

GLOBALLY replace "...blah, then" with "blah, and then" or "blah; then, "
A comma followed by "then" is not a conjunction.

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 33]
```diff
@@ -2,53 +2,40 @@
...
 == View Logs
 
-. In Runtime Manager, click *Applications* in the left menu, then click the application that is deployed to CloudHub to open its management options.
-. In the left menu, click *Logs*. The application logs are displayed.
-+
-//image::elmstart.png[ELMStart]
+View logs for applications deployed to CloudHub.
 
+. In Runtime Manager, click *Applications* in the left menu, then click the application that is deployed to CloudHub to open the management panel on the right. 
+. Click *Logs*. The application logs are displayed.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: Omit the second sentence or move it below the step as a subordinate paragraph.

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 34]
```diff
@@ -2,53 +2,40 @@
...
+Each worker in each CloudHub deployment has separate logs you can download.
 
-In the *Deployment* filters section on the right, find the date and worker you want to view, click the download icon next to it, and select *Logs*.
+. In the *Deployment* filters section on the right, find the date and worker you want to view. 
+. Click the download icon next to the worker, and select *Logs*.
 
 image::downloadlogs.png[DownloadLogs]
 
-The log file is a text file with `.log` file extension and named in the format of `<deployment-ID>-<instance-ID>.log`. You can view this file with a text editor.
+The log file is a text file with `.log` file extension and named in the format of `<deployment-ID>-<instance-ID>.log`. You can view the log file with a text editor.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: The log file is a text file with the format '<deployment... .log'.

The format provides the extension. But if it is important to call that out, then STET with the following slight adjustments: "...with the file extention .log, in the format '<deployment...."

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 35]
```diff
@@ -2,53 +2,40 @@
...
 
 [NOTE]
-If CloudHub application logs need to be archived or downloaded for audit, analytics or similar purpose on regular intervals, use xref:custom-log-appender.adoc[Custom Log Appender] to extract logs.
+If you need to archive or download CloudHub application logs for audit, analytics, or other purposes at regular intervals, use the xref:custom-log-appender.adoc[Custom Log Appender] to extract logs.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: "To archive or download...."

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 36]
```diff
@@ -68,7 +56,7 @@ The downloaded log file is a text file with the `.txt` file extension and named
 ====
 In diagnostics, the instance ID of a worker is the deployment ID plus the worker number (0 to 4).
 
-You can also download a diagnostics file directly, without having to go through the UI, by accessing the following URL:
+You can also access the following URL to download a diagnostics file directly:
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: You can also use....

I don't think one actually accesses a UR locator....

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 37]
```diff
@@ -151,8 +140,11 @@ Possible values are:
 You can filter log data by entering search terms into the search box at the top of the log page that match results in the log message. You can also search for an exact term in the log message with a few common quantifiers for wildcard searching:
 
 * `?` - The question mark matches zero or one of the preceding elements.
-* `*` - The asterisk matches zero or more of the preceding element.
+* `&#8727;` - The asterisk matches zero or more of the preceding element.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: element > elements

**Review by kbostrom — APPROVED** [review]
Looks great!

File: modules/ROOT/pages/managing-deployed-applications.adoc (hunk 28 above)
Comment (valkyrie69) [reply modules/ROOT/pages/managing-deployed-applications.adoc:288]: Text is necessary.

File: modules/ROOT/pages/managing-deployed-applications.adoc (hunk 30 above)
Comment (valkyrie69) [reply modules/ROOT/pages/managing-deployed-applications.adoc]: Customers is not correct. 
This is actually a self-service thing the user is doing for themselves to control what email alerts they themselves receive. 
So it probably still needs to be reworded, but not with "customer."

File: modules/ROOT/pages/viewing-log-data.adoc (hunk 32 above)
Comment (valkyrie69) [reply modules/ROOT/pages/viewing-log-data.adoc]: No semicolons please. They rarely belong in technical writing.

File: modules/ROOT/pages/managing-deployed-applications.adoc (hunk 6 above)
Comment (hannanelson) [reply modules/ROOT/pages/managing-deployed-applications.adoc:26]: Looks like a troubleshooting tool name.

File: modules/ROOT/pages/viewing-log-data.adoc (hunk 32 above)
Comment (hannanelson) [reply modules/ROOT/pages/viewing-log-data.adoc]: Interesting. My previous editor would disagree with the ", and then" construction - she'd say either the and or the then :)
